### PR TITLE
fix: add missing secrets for requirements job

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
**Description:**
This PR adds missing SMTP secrets for the requirements job, which [failed](https://github.com/openedx/openedx-events/actions/runs/3024132560) while setting up the environment. This is a continuation of PR #106 since we missed those secrets there.

**JIRA:** 
None

**Dependencies:** 
None

**Merge deadline:** 
ASAP

**Installation instructions:** 
None, it's a github action change.

**Testing instructions:**
1. Go to actions
2. Select this branch
3. Select upgrade requirements job
4. Run job
You'll pass the setup step, & it will fail while creating the pull request.

**Reviewers:**
- [x] @felipemontoya 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** 
The job fails with 404 while creating the pull request, more info [here](https://github.com/openedx/openedx-events/runs/8275216199?check_suite_focus=true#step:7:54)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mariajgrimaldi!

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

**Review by felipemontoya — APPROVED** [review]
Oh boy, so this was it. The 404 came at the smtp message time and not at the PR time.

**Comment by openedx-webhooks** [comment]
: thought you might like to know that mariajgrimaldi merged this pull request.

**Comment by openedx-webhooks** [comment]
@mariajgrimaldi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-events&entry.867055334=https://github.com/openedx/openedx-events/pull/108&entry.1484655318=https://github.com/mariajgrimaldi&entry.752974735=2022-09-09+17:11&entry.1917517419=2022-09-12+17:34&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
